src/csharp/Grpc.Core/Internal/UnmanagedLibrary.cs

```diff
@@ -114,6 +114,10 @@ public IntPtr LoadSymbol(string symbolName)
                 {
```

Comment:
nit: there's a comment at the top of this file talking about PInvoke, since we don't use PInvoke after https://github.com/grpc/grpc/pull/7401/files. Missed this earlier, but maybe should change it.